src/_tutorials/server/get-started.md

```diff
@@ -44,24 +44,18 @@ More information:
 
 {% include get-sdk.md %}
 
-<!-- PENDING: the following instructions assume you have set the PATH.
-     We should update the included instructions to refer to that. -->
-
 ## 3. Get more command-line developer tools
 
 Install [`stagehand`,][stagehand] which gives you templates for creating Dart apps:
 
 ```terminal
-> pub global activate stagehand
+$ pub global activate stagehand
 ```
 
-
 Note that although these instructions feature the command line,
 many IDEs support Dart development.
 Those IDEs use Stagehand behind the scenes when you create new Dart projects.
 
-<!-- PENDING: the following instructions assume you have the bin directory for the system cache in your path. -->
```

Comment:
Adding the system cache bin to one's path is usually a onetime setup. The "More information" bullet [Running a script from your path](http://localhost:4000/tools/pub/cmd/pub-global#running-a-script-from-your-path) explains how to adjust `PATH`.

Is that good enough? Or should we convert the "More info" bullet into a warning?

Reply:
I think we're good.